.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+# We do not use Github for issues!
+
+Please do not create issues here.
+
+## To get help
+
+If you need help using [Archive of Our Own](http://archiveofourown.org), please [contact its Support team](http://archiveofourown.org/support)
+
+## To suggest a new feature
+
+[AO3 Support](http://archiveofourown.org/support) also track feature requests.
```

Comment:
I'll change that to say "The AO3 Support team" as in my head, "AO3 Support" is potentially a collective noun, and "team" is unambiguous.